app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -94,8 +95,7 @@ $font-family-sans-serif: "Nixie One", "Helvetica Neue", Helvetica, Roboto, Arial
 // We use these to define default font weights
 // $font-weight-normal: normal;
 // $font-weight-bold: bold;
-
-// $white       : #FFFFFF;
+$white       : #FFFFFF;
```

Comment:
Color `#FFFFFF` should be written as `#ffffff`<br>Variable names should be followed immediately by a colon